pkg/lib/machine-dialogs.js

```diff
@@ -147,6 +149,7 @@
                 conn_options['temp-session'] = false; /* Compatiblity option */
                 conn_options['session'] = 'shared';
                 conn_options['host-key'] = machine.host_key;
+                conn_options['unknown-host'] = "1";
```

Comment:
Ah, and the `COCKPIT_SSH_ALLOW_UNKNOWN=${unknown-host}` bridge match is the vehicle in this case. Thanks for the explanation!